engine/table/src/main/java/io/deephaven/engine/table/impl/NaturalJoinHelper.java

```diff
@@ -988,7 +988,7 @@ protected void process() {
                         // We are almost able to fix this, because we know the hash slot and the result redirection for
                         // the left modified row; which is the new value.
                         // We could get the hash slot from the removal, and compare them, but the hash slot outside of a
-                        // modified slot tracker is unstable [and we don’t want two of them].
+                        // modified slot tracker is unstable [and we don't want two of them].
```

Comment:
Do single quotes work? I'm assuming you generated the JavaDoc and checked it out?

Reply:
This is just a comment, not javadoc, so I don't think it matters.  The original char one caused an illegal char error during compile

Reply:
Duh, sorry